scripts/apps/authoring/authoring/controllers/AssociationController.ts

```diff
@@ -7,7 +7,7 @@ import {mediaIdGenerator} from '../services/MediaIdGeneratorService';
 
 export function getAssociationsByField(item: IArticle, field: IVocabulary) {
     return Object.keys(item.associations || {})
-        .filter((key) => key.startsWith(field._id) && item.associations[key] != null)
+        .filter((key) => key.startsWith(field?._id) && item.associations[key] != null)
```

Comment:
Can a vocabulary not have `_id`? When?

Reply:
Good point, actually in case of featuremedia we don't have field itself as it's not a vocabulary. I will fix this.